scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -749,14 +750,34 @@ def validate_model_spec(asset_config: assets.AssetConfig) -> int:
         _log_error(asset_config.file_name_with_path, f"{MLFlowModelTags.LICENSE} missing")
         error_count += 1
 
+    # shared compute check
     if MLFlowModelTags.SHARED_COMPUTE_CAPACITY not in model.tags:
-        _log_error(asset_config.file_name_with_path, f"{MLFlowModelTags.SHARED_COMPUTE_CAPACITY} missing")
+        _log_error(asset_config.file_name_with_path, f"Tags {MLFlowModelTags.SHARED_COMPUTE_CAPACITY} missing")
+        error_count += 1
+
+    if not model.properties.get(MLFlowModelProperties.SHARED_COMPUTE_CAPACITY, False):
+        _log_error(
+            asset_config.file_name_with_path, f"Property {MLFlowModelProperties.SHARED_COMPUTE_CAPACITY} is not set"
+        )
         error_count += 1
 
-    supports_eval = model.tags.get(MLFlowModelTags.EVALUATION_COMPUTE_ALLOWLIST) is not None
-    supports_ft = model.tags.get(MLFlowModelTags.FINETUNE_COMPUTE_ALLOWLIST) is not None
+    # if any of the relevant tags or properies is present
+    # assume support as true and then fail in valdn
+    supports_eval = (
+        model.tags.get(MLFlowModelTags.EVALUATION_COMPUTE_ALLOWLIST) is not None
+        or model.properties.get(MLFlowModelProperties.EVALUATION_RECOMMENDED_SKU) is not None
+        or model.properties.get(MLFlowModelProperties.EVALUATION_MIN_SKU_SPEC) is not None

```

Comment:
When you don't need to retrieve the values, please use `in` tests for these keys instead of `get`...`is not None`. For example, `MLFlowModelTags.EVALUATION_COMPUTE_ALLOWLIST in model.tags`.

Reply:
Done, thanks Louie